packages/dds/merge-tree/src/client.ts

```diff
@@ -245,7 +247,7 @@ export class Client {
         this.completeAndLogOp(
             opArgs,
             this.getClientSequenceArgs(opArgs),
-            { start: op.pos1, end: undefined },
+            { start: op.pos1!, end: -1 },
```

Comment:
can you just change completeAndLogOp to take a Partial<IIntegerRange>, rather than defaulting

Reply:
done.